src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Interop/User32.cs

```diff
@@ -0,0 +1,1632 @@
+        /// </summary>
+        public IntPtr hCursor;
+
+        /// <summary>
+        /// A handle to the class background brush. This member can be a handle to the brush to be used for painting the background, or it can be a color value.
+        /// </summary>
+        public IntPtr hbrBackground;
+
+        /// <summary>
+        /// Pointer to a null-terminated character string that specifies the resource name of the class menu, as the name appears in the resource file. If you use an integer to identify the menu, use the MAKEINTRESOURCE macro. If this member is NULL, windows belonging to this class have no default menu.
+        /// </summary>
+        [MarshalAs(UnmanagedType.LPWStr)]
+        public string lpszMenuName;
+
+        /// <summary>
+        /// A pointer to a null-terminated string or is an atom. If this parameter is an atom, it must be a class atom created by a previous call to the RegisterClass or RegisterClassEx function. The atom must be in the low-order word of lpszClassName; the high-order word must be zero.
+        /// </summary>
+        [MarshalAs(UnmanagedType.LPWStr)]
+        public string lpszClassName;
+
+        /// <summary>
+        /// A handle to a small icon that is associated with the window class. If this member is NULL, the system searches the icon resource specified by the hIcon member for an icon of the appropriate size to use as the small icon.
+        /// </summary>
+        public IntPtr hIconSm;
+    }
+
+    /// <summary>
+    /// Delegate declaration that matches native WndProc signatures.
+    /// </summary>
+    public delegate IntPtr WndProc(IntPtr hWnd, WM uMsg, IntPtr wParam, IntPtr lParam);
+
+    /// <summary>
+    /// Delegate declaration that matches native WndProc signatures.
+    /// </summary>
+    public delegate IntPtr WndProcHook(IntPtr hWnd, WM uMsg, IntPtr wParam, IntPtr lParam, ref bool handled);
+
+    /// <summary>
+    /// Delegate declaration that matches managed WndProc signatures.
+    /// </summary>
+    public delegate IntPtr MessageHandler(WM uMsg, IntPtr wParam, IntPtr lParam, out bool handled);
+
+    /// <summary>
+    /// The ReleaseDC function releases a device context (DC), freeing it for use by other applications.
+    /// The effect of the ReleaseDC function depends on the type of DC. It frees only common and window DCs. It has no effect on class or private DCs.
+    /// </summary>
+    /// <param name="hWnd">A handle to the window whose DC is to be released.</param>
+    /// <param name="hDC">A handle to the DC to be released.</param>
+    /// <returns>The return value indicates whether the DC was released. If the DC was released, the return value is 1. If the DC was not released, the return value is zero.</returns>
+    [DllImport(Libraries.User32, CharSet = CharSet.Auto)]
```

Comment:
I think we should set the CharSet as `CharSet.Unicode` by default.

Reply:
Can we consider using GeneratedDllImport for win32 Interop code in the new code? See https://github.com/dotnet/wpf/issues/7040